src/android/notification/ClickActivity.java

```diff
@@ -40,12 +40,6 @@
     @Override
     public void onClick(Notification notification) {
         launchApp();
-
-        if (notification.isRepeating()) {
-            notification.clear();
-        } else {
-            notification.cancel();
```

Comment:
Ugh, I wish I remembered. I think that was back when I was trying to do things a different way, and it  never got moved back. So, not good reason, just my sloppiness. Sorry about that! I've fixed the this locally.